seleniumbase/core/browser_launcher.py

```diff
@@ -1318,36 +1315,12 @@ def get_remote_driver(
                 keep_alive=True,
             )
     elif browser_name == constants.Browser.REMOTE:
-        selenoid = False
-        selenoid_options = None
-        screen_resolution = None
-        browser_version = None
-        platform_name = None
-        for key in desired_caps.keys():
-            if key == "selenoid:options":
-                selenoid = True
-                selenoid_options = desired_caps[key]
-            elif key == "screenResolution":
-                screen_resolution = desired_caps[key]
-            elif key == "version" or key == "browserVersion":
-                browser_version = desired_caps[key]
-            elif key == "platform" or key == "platformName":
-                platform_name = desired_caps[key]
+        selenoid = "selenoid:options" in desired_caps
```

Comment:
this looks unused? if the user has provided a `selenoid:options` key it will be shoveled in under the same key - ok to omit?